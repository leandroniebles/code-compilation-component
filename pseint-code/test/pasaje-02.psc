subproceso lala(b por referencia)
FinSubProceso
Proceso sin_titulo
	lala(2);
	lala(x);
	lala(x+2);
	lala(w[2]);
	lala(azar(2));
FinProceso
