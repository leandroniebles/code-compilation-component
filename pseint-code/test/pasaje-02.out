=== Line 4: SynError 268
=== Line 6: SynError 268
=== Line 8: SynError 268
